.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,21 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
Please add a README file in this dir (or upstream) to indicate what this config is for and where the spec is defined.

Reply:
Done. I just added some notes on how this is created but did not bother to link to IntelliJ documentation for the spec (because it is not hand made).

Reply:
Sounds good. Thanks!